### DISCUSSION TITLE
Question regarding ViewLocator usage

### DISCUSSION THREAD

**Discussion by Swiftpaws** [reactiveui/ReactiveUI#2750] [Q&A]
![image](https://user-images.githubusercontent.com/18440081/117407633-1c5a9200-af0f-11eb-8248-e64384b27a8e.png)

The documentation is not really going into detail on how to resolve views when you are not using a Router, but from what I gathered this should? work because there is a Default ViewLocator in place.

**Comment by glennawatson** [comment]
The first call into most ReactiveUI systems should register this the DefaultViewLocator

https://github.com/reactiveui/ReactiveUI/blob/740ae1a5b3d219dcf62c944ef44e7dbdf38c7d26/src/ReactiveUI/Registration/Registrations.cs#L33

It appears as the dependency isn't getting registered in your case. Are you using any custom dependency injection layers?

**Reply by Swiftpaws** [reply]
No, this is from a test project using .net 4.8 (Also tested with .net 5.0)
Nuget:
![image](https://user-images.githubusercontent.com/18440081/117435788-87679100-af2e-11eb-89ed-8d7083d3bb44.png)
